go/libraries/doltcore/mvdata/csv_helper.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mvdata
+
+import (
+	"github.com/dolthub/dolt/go/libraries/doltcore/table/untyped/csv"
+)
+
+// CreateCSVInfo creates a CSVInfo object based on the provided options.
+// This is a helper function that extracts and processes CSV-related options
+// from the generic options interface.
+func CreateCSVInfo(opts interface{}, defaultDelim string) *csv.CSVFileInfo {
```

Comment:
not sure how y'all feel about this file name. Wasn't sure if I should name it after helper func, or keep it a generic helper file. 🤷‍♂️